exif2timestream.py

```diff
@@ -9,8 +9,10 @@
 from voluptuous import Required, Schema, MultipleInvalid
 from itertools import cycle
 from inspect import isclass
+from PIL import Image
```

Comment:
Or use scikit image. We use scikit image everywhere else so it should be installed.